brianchandotcom/liferay-portal
# LPS-167023 Move Segmentation from People to the Control Panel.

Motivation
=======
In order to Unify Segmentation, we need to move the Segmentation Portlet Access from Site level to the Control Menu.

The reason to do that is to support Commerce and AC use case.

[Link to story or ticket](https://issues.liferay.com/browse/LPS-167023)

Proposed Solution
========

Once the FF LPS-167023 is set to true SegmentsPanelApp is moved from Site >  People > Segments to Control Panel > Users > Segments

We also have need to change the way we retrieve Segments in view.jsp, as now all the Segments of the company will be visible.

The new service methods have been created in: https://github.com/liferay-tango/liferay-portal/pull/3041

Steps to Verify:
----------------
1. Set feature.flag.LPS-166954=true in portal-ext
1. Go to Control Panel > Users > Segmentation
1. The Segments can be created without issues.

Screenshots:
-----------------------
![Captura de pantalla 2022-11-16 a las 10 02 04](https://user-images.githubusercontent.com/5776822/202150391-07ef067e-47a4-417f-b2aa-0af54fb79852.png)

![Captura de pantalla 2022-11-18 a las 11 21 37](https://user-images.githubusercontent.com/5776822/202694676-73a82318-9e91-43dd-9b96-abd7def42881.png)


Note:
-----------------------
The included tests, are general, tests, since we have the FF, and the default behavior won't last long. Once we delete the FF, the test will test that the Portlet is displayed in the right section.

Original PR: https://github.com/liferay-tango/liferay-portal/pull/3026
Failing tests not related with this PR.
